webapp/action_creators/websocket_actions.jsx

```diff
@@ -147,6 +152,7 @@ export function sendMessage(msg) {
 }
 
 export function close() {
+    manuallyClosed = true;
```

Comment:
It's called in the unmount of logged_in.jsx https://github.com/mattermost/platform/blob/master/webapp/components/logged_in.jsx#L177 which will occur on logout and also when the window unloads https://github.com/mattermost/platform/blob/master/webapp/root.jsx#L110